aiven/client/cli.py

```diff
@@ -3823,6 +3823,8 @@ def vpc__user_peer_network_cidr__delete(self) -> None:
         default=False,
         help="Enable termination protection",
     )
+    @arg("--service-to-fork-from", help="Service to fork from")
+    @arg("--recovery-target-time", help="PITR recovery point in format 2023-02-01 11:38:49")
```

Comment:
```suggestion
    @arg("--recovery-target-time", help="PITR recovery point datetime in ISO 3166 format")
```

Reply:
I disagree with this. A concrete example tells me immediately what to do, no need to force the user to study standards.